composer.json

```diff
@@ -31,25 +31,24 @@
     "require": {
         "php": ">=7.4.3,<8.2.0",
         "ext-json": "*",
-        "guzzlehttp/guzzle": "^7.4.1",
-        "league/flysystem": "^2.3.2 || ^3.0.0",
+        "guzzlehttp/guzzle": "^7.4.2",
+        "league/flysystem": "^2.4.5 || ^3.0.18",
         "matthiasmullie/scrapbook": "^1.4.8",
-        "monolog/monolog": "^2.3.5",
+        "monolog/monolog": "^2.5.0",
```

Comment:
Monolog 3 released: https://github.com/Seldaek/monolog/releases/tag/3.0.0

Reply:
Thesre are incompatible changes which do not work for PHP 7.4.